lib/src/document.dart

```diff
@@ -65,6 +66,8 @@ class Document {
   List<Node> parseLines(List<String> lines) {
     final nodes = BlockParser(lines, this).parseLines();
     _parseInlineContent(nodes);
+    // after parsing inline as we need ref count
```

Comment:
Make sure to start comments with a capiital, and end with a period. https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences

Reply:
done